### ISSUE TITLE
Adding zoom preview feature

### ISSUE THREAD

**Issue by traverseda** [drwhut/tabletop-club#66]
So I'm trying to add in a zoom-preview feature, because it seems like a good way to get myself familiar with the general layout of this project. I'm going to need some guidance on this though, as I've never worked with godot before.

1. How do I have the keybindings? I've created a menu item in the options menu, and added the variable `game_zoom_preview`. I need to add a default, and have it actually save the bindings when they're changed.
2. Where would I put the code to handle the `game_zoom_preview` event? In `Game.gd`?
3. Can you give me a rough outline of what I'd need to do in order to figure out what asset is under the mouse and draw it larger in an overlay? Figuring out how to do this would also let me create previews of hands similar to how tabletop simulator does it.

Not that I'm going to get time to jump into it for a while...

**Comment by drwhut** [comment]
1. The default value for options in the options menu is the default value of the `Control` node associated with it. For keybindings, you'll need to go to `Project` > `Project Settings` > `Input Map` to create a key binding with a name, then use the custom key binding node I made, and give the name of the key binding to that node. If it's in the key bindings tab, the code *should* automatically save the key binding in the config file.

For points 2 and 3, I've already got functionality similar to what you're looking for, around the `HandPreviewRect` node and `_hand_preview_rect` variable in `CameraController.gd`. Have a look at that to see how to inspect the asset under the mouse, and potentially how to display it as well!
